data/human/weapons.txt

```diff
@@ -226,6 +226,29 @@ outfit "Modified Blaster Turret"
 		"hull damage" 8.8
 	description "This is a turreted version of the Modified Blaster, which is an Energy Blaster modified for greater power at the cost of higher energy consumption and heat. These turrets are popular with pirates and others who are trying to cram as much firepower into their ships as possible."
 
+outfit "Heavy Blaster"
+	category "Guns"
+	series "Guns"
+	index 01050
+	cost 140000
```

Comment:
```suggestion
	cost 92000
```
140000 is a bit expensive for a blaster, 92k puts it just above a twin mod blaster.

Reply:
This is quite a bit bigger than a twin mod blaster, though (19t vs 14t). It's almost as close to the plasma cannon (25t, 190k cost).

Reply:
I was mostly comparing it to the regular blaster, thinking of it as a scaled up version, it does ~4x the damage in ~4x the space, so a pricetag close to 6x seems pretty reasonable. Additionally, 140k seems too close to the proton gun's price, which is a way more powerful weapon.

Reply:
114k would give it the same cost per ton as the twin modified blaster. Given that cost per ton seems generally to go up at least a little with size, you could maybe put it in the 120k-130k range?
```suggestion
	cost 125000
```